src/appkit.rs

```diff
@@ -770,37 +777,37 @@ impl NSWindow for id {
     // Getting Layout Information
 
     unsafe fn contentRectForFrameRect_styleMask_(self, windowFrame: NSRect, windowStyle: NSUInteger) -> NSRect {
-        msg_send_stret()(self, selector("contentRectForFrameRect:styleMask:"), windowFrame, windowStyle)
+        msg_send![self, contentRectForFrameRect:windowFrame styleMask:windowStyle]
     }
 
     unsafe fn frameRectForContentRect_styleMask_(self, windowContentRect: NSRect, windowStyle: NSUInteger) -> NSRect {
-        msg_send_stret()(self, selector("frameRectForContentRect:styleMask:"), windowContentRect, windowStyle)
+        msg_send![self, frameRectForContentRect:windowContentRect styleMask:windowStyle]
     }
 
     unsafe fn minFrameWidthWithTitle_styleMask_(self, windowTitle: id, windowStyle: NSUInteger) -> CGFloat {
-        msg_send()(self, selector("minFrameWidthWithTitle:styleMask:"), windowTitle, windowStyle)
+        msg_send![self, minFrameWidthWithTitle:windowTitle styleMask:windowStyle]
     }
 
     unsafe fn contentRectForFrameRect_(self, windowFrame: NSRect) -> NSRect {
-        msg_send_stret()(self, selector("contentRectForFrameRect:"), windowFrame)
+        msg_send![self, contentRectForFrameRect:windowFrame]
     }
 
     unsafe fn frameRectForContentRect_(self, windowContent: NSRect) -> NSRect {
-        msg_send_stret()(self, selector("frameRectForContentRect:"), windowContent)
+        msg_send![self, frameRectForContentRect:windowContent]
```

Comment:
Good catch; I've been meaning to do another pass on the documentation. Updated this one in ssheldon/rust-objc@67c2399.

Does servo need to support 32bit OSX? Haven't published it yet since I don't have a way to test, but I've looked into adding support: ssheldon/rust-objc@79d33ba.